d2renderers/d2latex/latex.go

```diff
@@ -0,0 +1,79 @@
+//go:build cgo
+
+package d2latex
+
+import (
+	_ "embed"
+	"fmt"
+	"math"
+	"regexp"
+	"strconv"
+
+	"oss.terrastruct.com/xdefer"
+	v8 "rogchap.com/v8go"
+)
+
+//go:embed polyfills.js
+var polyfillsJS string
+
+//go:embed setup.js
+var setupJS string
+
+//go:embed mathjax.js
+var mathjaxJS string
+
+var svgRe = regexp.MustCompile(`<svg[^>]+width="([0-9\.]+)ex" height="([0-9\.]+)ex"[^>]+>`)
```

Comment:
Maybe just an example that it matches `width=".." height=".."`

Reply:
is it true that `width` will always come before `height` and nothing will be in between?

Reply:
i haven't verified in MathJax's source code, but I see no reason why it'd change